tests/integration/thrift/test_tornado_client.py

```diff
@@ -82,7 +83,7 @@ def test_thrift_exception(mock_server, thrift_service):
         'thrift',
         method='getItem',
     ).and_raise(thrift_service.ItemDoesNotExist("stahp"))
-
+    mock_server.tchannel.hooks.register(ZipkinTraceHook(tchannel=mock_server))
```

Comment:
Existing tests may have been asserting zipkin behavior because we had it enabled by default previously.

Now that it's opt-in, it might make sense to eliminate these references.
Although, we would still want an integration test of some kind